pkg/minikube/machine/cache_images.go

```diff
@@ -660,6 +660,9 @@ func RemoveImages(images []string, profile *config.Profile) error {
 
 	klog.Infof("succeeded removing from: %s", strings.Join(succeeded, " "))
 	klog.Infof("failed removing from: %s", strings.Join(failed, " "))
+	if len(failed) > 0 {
+		return errors.Errorf("failed nodes: %s, add --alsologtostderr option to get more messages.", strings.Join(failed, " "))
```

Comment:
We should output the actual error without the user having to run `--alsologtostderr`.

I believe if you change the `klog.Warningf` on line 654 from `Warningf` to `Errorf` it will also output the error to the user.

Reply:
I agree, lets keep the returned error Short
and instead add klog.Warnining("failed to remove image %s", x)

Reply:
ok, I will change the code.